index.html

```diff
@@ -1163,9 +1163,13 @@ <h4 id="language-detection-issues">Issues</h4>
 <section>
 	<h2 id="localization-considerations">Localization Considerations</h2>
 	
-	<p>Many specifications need to allow multiple different language values to be returned for a given field. This might be to support runtime localization or because the <a>producer</a> has multiple different language values and cannot select or distinguish them appropriately. There are several ways that multiple language values could be organized. For speed and ease of access, the use of <dfn data-lt="language indexing">language indexing</dfn> is a useful strategy.</p>
+	<p>Sometimes a <a>producer</a> can supply localized values for a given content item or data record by performing some type of <a>language negotiation</a> between the <a>producer</a> and the <a>consumer</a>. Localization then takes place in the <a>producer</a> using the negotiated language to select the content returned. Such an approach can save on file size, which affects latency, and complexity, since only the language or languages needed by the <a>consumer</a> need be returned.</p>
 	
-	<p>In language indexing, a given field's value is an array of key-value pairs. The keys in the array are language tags. The values of each language tag are strings or, ideally, <a>Localizable</a> objects. Here's an example of what a language indexed field <kbd>title</kbd> might look like:</p>
+	<p>However, since this is not always possible, specifications sometimes allow multiple different language values to be returned for a given field. This might be to support runtime localization or because the <a>producer</a> has multiple different language values and cannot pre-select them appropriately.</p>
+	
+	<p>In these cases, localization of a content item is done by having the <a>producer</a> return multiple language representations for the item and letting the <a>consumer</a> choose the value to display. Such an approach is helpful when the <a>producer</a> cannot negotiate the language (such as when the resulting file is cached for multiple users) and when the number of languages is relatively small. Large collections of languages can result in overly large documents that are cumbersome to work with.</p>
+	
+	<p>One approach a specification might provide for returning mulitple language of a given field is called <dfn>language indexing</dfn>. In language indexing, a given field's value is an array of key-value pairs. The keys in the array are language tags. The values of each language tag are strings or, ideally, <a>Localizable</a> objects. Here's an example of what a language indexed field <kbd>title</kbd> might look like:</p>
```

Comment:
mulitple -> multiple

Reply:
Shouldn't that be 'multiple languages' (with an -s) ?

Reply:
🤦 See #79. Thanks for spotting these.